### PR TITLE
QA-191: Add a PR contributor template

### PR DESCRIPTION
**Pull request by oleorhagen**
Changelog: None
Signed-off-by: Ole Petter <ole.orhagen@northern.tech>

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+
+# External Contributor Checklist
+
+🚨 Please review the [guidelines for contributing](../CONTRIBUTING.md) to this repository.
+
+- [ ] Make sure that all commits have a [`Changelog`](../CONTRIBUTING.md#changelog-tags) tag. If nothing should be added to the Changelog, add a `Changelog: None` tag. If there is a change, add `Changelog: Commit`, or `Changelog: Title`, depending on what should be included in the changelog.
```
Comment (lluiscampos) [inline .github/PULL_REQUEST_TEMPLATE.md:6]: There is no changelog requirement in this repo

Comment (oleorhagen) [reply .github/PULL_REQUEST_TEMPLATE.md:6]: Hmm, good point. And no `CONTRIBUTING.md` it seems. Should we add this file along with this?

Comment (lluiscampos) [reply .github/PULL_REQUEST_TEMPLATE.md:6]: Oh. I didn't realize that the others were also pointing to the local `CONTRIBUTING.md`. All repos should point to `mender/CONTRIBUTING.md`.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+
+# External Contributor Checklist
+
+🚨 Please review the [guidelines for contributing](https://github.com/mendersoftware/mender/blob/master/CONTRIBUTING.md) to this repository.
```
Comment (lluiscampos) [inline .github/PULL_REQUEST_TEMPLATE.md:4]: While I was commenting on the link you fixed it :smile:  :+1:

**Pull request merged**